contracts/ERC721A.sol

```diff
@@ -596,4 +601,44 @@ contract ERC721A is Context, ERC165, IERC721A {
         uint256 startTokenId,
         uint256 quantity
     ) internal virtual {}
+
+    /**
+     * @dev Returns the message sender (defaults to `msg.sender`).
+     */
+    function _msgSender() internal view virtual returns (address) {
+        return msg.sender;
+    }
+
+    /**
+     * @dev Returns the message data (defaults to `msg.data`).
+     */
+    function _msgData() internal view virtual returns (bytes calldata) {
+        return msg.data;
+    }
+
+    /**
+     * @dev Converts a `uint256` to its ASCII `string` decimal representation.
+     */
+    function _toString(uint256 value) internal pure returns (string memory) {
```

Comment:
Might be worth throwing a test or two for this into our test suite

Reply:
Already added.